website/frontend/templates/docs/scripting.html

```diff
@@ -28,20 +28,75 @@ <h1 id="metadata_variables">Metadata Variables</h1>
 
       <p>
         See <a href="/docs/tags/" title="Picard Tags">Picard tags</a> for
-        list of usable variables.
+        list of standard variables provided by Picard.
+      </p>
+
+      <p>
+        Picard's standard variables can be either simple variables containing a single text string,
+        or <strong>multi-value</strong> variables containing multiple text strings.
+        In scripts, multi-value variables are automatically converted to a single text string
+        by joining the values with <code>; </code>,
+        <strong>except</strong> when used with special multi-value functions.
       </p>
 
       <div id="scripting_technical_variables">
 
         <h1 id="technical_variables">Technical Variables</h1>
```

Comment:
I think it's better to remove this section because it only duplicates https://picard.musicbrainz.org/docs/tags/#basic_variables